client/admin.py

```diff
@@ -0,0 +1,3 @@
+# from django.contrib import admin
```

Comment:
register your module so you can visualize it with admin UI

Reply:
thanks, just did it